lib/authentication.js

```diff
@@ -40,6 +40,11 @@ module.exports = {
                next(err);
```

Comment:
Hmm.. shouldn't this be `return next()`? Maybe that's the source of the problem?